62/serfer2/scraper.py

```diff
@@ -0,0 +1,71 @@
+# coding=utf-8 -*-
+
+import requests
+from lxml import html
+
+SPLIT_SYMBOLS = [',', ' y ', ' e ', ' - ']
+
+
+class NameScraper(object):
+    """
+    Scrapping names for a given url and xpath
+    """
+
+    _url: str = None
+    _xpath: str = None
+
+    def __init__(self, config):
+        self._url = config['url']
+        self._xpath = config['xpath']
+
+    def get_names(self) -> list:
+        """
+        get all found names
+        """
+        speakers = self.get_items_from_url()
+        for symbol in SPLIT_SYMBOLS:
+            speakers = self.split_names(speakers, symbol)
```

Comment:
this seems not very Pythonic...by first reading over it I thought this can't be right as speakers is overwritten by split_names return value several times but because speakers is always the last returned value of this function this is correct in this context....but as I said it doesn't read nicely and is not obvious what is done here...so you pass the same input several times and split with different symbols. So if you need to split at multiple characters use `re.split` method which allows the definition of multiple characters with "or"